ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -160,8 +196,16 @@ objects:
           # TODO: make openshift-tests auto-discover this from cluster config
           export TEST_PROVIDER='{"type":"aws","region":"us-east-1","zone":"us-east-1a","multizone":true,"multimaster":true}'
           export KUBE_SSH_USER=core
+          if [[ "${ENABLE_FIPS}" == true ]]; then
```

Comment:
why is this in this block?  Move it down out of this block.

Reply:
Moved down out of the if blocks